docs/reference.md

```diff
@@ -114,3 +114,64 @@ Message communication example
 |                                    | ← {"source":"custom_embed","action":"data","key":"j0a","data":{…}}                                        |
 |                                    | /or/ User cancelled changes                                                                   |
 |                                    | ← {"source":"custom_embed","action":"cancel","key":"j0a"}                                                 |
+
+# Request additional Story ANS
+Customers can obtain addition ANS data from Ellipsis via `postMessage`. Ellipsis/Composer will send back the data with the additional fields of:
+
+1. Headlines
+2. Subheadlines
+3. Taxonomy
+
+To acheive this, simply
+- Add an additional flag `isAnsRequired` set to `true` in your API where `postMessage()` is happening. E.g.
```

Comment:
I would prefer this to be specified for `Ready message` only

Reply:
Composer will only send message when `action = 'ready'`, therefore I believe that we already have it covered.

Reply:
This is the documentation for our clients not for us.
They should know the details